quickwit/quickwit-config/src/node_config/mod.rs

```diff
@@ -55,6 +55,8 @@ pub struct IndexerConfig {
     pub enable_otlp_endpoint: bool,
     #[serde(default = "IndexerConfig::default_enable_cooperative_indexing")]
     pub enable_cooperative_indexing: bool,
+    #[serde(default = "IndexerConfig::default_capacity")]
+    pub capacity: u32,
```

Comment:
I think `capacity` is too ambiguous. Also, I'd rather expose our users to the pipeline concept as little as possible. I think we should ask our users how many CPUs they want to allocate to their indexers, and we infer how many pipelines we can pack from there. With that in mind, `cpu_capacity` is probably what we want.

https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes

Reply:
You are right. I introduced a CpuCapacity. We can now configure it either as a string "1000m" or as a float. (1.0)